docs/source/rpc.rst

```diff
@@ -231,6 +189,48 @@ Example::
     :inherited-members:
 
 
+Process Group Backend
+"""""""""""""""""""""
+
+The Process Group agent instantiates a process group from
+the :mod:`~torch.distributed` module and utilizes its point-to-point
+communication capabilities to send RPC messages across. Internally, the process
```

Comment:
nit: we can probably remove the word "across"

Reply:
edited in the PR on top.